### ISSUE TITLE
Emacs30 version out of date by one month

### ISSUE THREAD

**Issue by LemonBreezes** [nix-community/emacs-overlay#382]
Can we have a derivation for the latest daily builds? The current `emacs-git` derivation is already one month behind `master`.

**Comment by LemonBreezes** [comment]
Okay, so I see that this is related to 
https://github.com/NixOS/nixpkgs/pull/276943
so I suppose we're just waiting on that.

**Comment by leungbk** [comment]
The latest Emacs 30 should work on the latest `nixos-unstable` with the recent `staging-next` being merged into `master`, but we should wait for `23.11` to get updated as well since I expect that there are some people who use both `23.11` and Emacs 30.
